inception-imls-opennlp/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/doccat/OpenNlpDoccatRecommender.java

```diff
@@ -226,18 +224,18 @@ public EvaluationResult evaluate(List<CAS> aCasses, DataSplitter aDataSplitter)
                 
                 Type annotationType = getType(cas, layerName);
                 Feature feature = annotationType.getFeatureByBaseName(featureName);
+                Collections.addAll(annotations, selectCovered(annotationType, sentence));
+            }
+            for (AnnotationFS annotation : selectCovered(annotationType, sentence)) {
```

Comment:
You are not using the collection `annotations` which you have populated in the first loop. Please write test cases which would find such faults.

Reply:
There is an error in this loop, I want to reduce the loop statement but it will cause problems.